pkg/authorization/authorizer/attributes.go

```diff
@@ -20,6 +20,16 @@ type DefaultAuthorizationAttributes struct {
 	URL               string
 }
 
+// ToDefaultAuthorizationAttributes coerces AuthorizationAttributes to DefaultAuthorizationAttributes.  Namespace is not included
+// because the authorizer takes that information on the context
```

Comment:
> if you submit a cluster-scoped subjectaccessreview, the context has no namespace... won't this drop the namespace specified in the authorization attributes?

`DefaultAuthorizationAttributes` has no namespace because it's included in the `Context`.  I guess I could accept and return a context if you liked.